cmd/opampsupervisor/supervisor/templates/owntelemetry.yaml

```diff
@@ -7,4 +7,6 @@ service:
             exporter:
               otlp:
                 protocol: http/protobuf
-                endpoint: {{.MetricsEndpoint}}
+                endpoint: {{.MetricsEndpoint}}{{if .MetricsHeaders}}
+                headers:{{range $k := .MetricsHeaders}}
+                  "{{$k.Key}}": "{{$k.Value}}"{{end}}{{end}}
```

Comment:
Maybe I'm too used to helm, but this template feels off.  I expected 

```yaml
endpoint: {{.MetricsEndpoint}}
{{- if .MetricsHeaders}}
headers:
{{- range $k := .MetricsHeaders}}
  "{{$k.Key}}": "{{$k.Value}}"
{{- end}}
{{- end}}
```

Reply:
updated as per the suggestion